sdk/scheduler/src/test/java/com/mesosphere/sdk/scheduler/plan/DefaultPlanCoordinatorTest.java

```diff
@@ -159,6 +157,43 @@ public void testOnePlanManagerPendingSufficientOffer() throws Exception {
     }
 
     @Test
+    public void testPodInstanceRequirementConflictsWhenTasksOverlap() {
```

Comment:
Are there test cases for the other requirement that `conflictsWith` JavaDoc lists?